openstack/networking/v2/ports/requests.go

```diff
@@ -17,19 +17,20 @@ type ListOptsBuilder interface {
 // by a particular port attribute. SortDir sets the direction, and is either
 // `asc' or `desc'. Marker and Limit are used for pagination.
 type ListOpts struct {
-	Status       string `q:"status"`
-	Name         string `q:"name"`
-	AdminStateUp *bool  `q:"admin_state_up"`
-	NetworkID    string `q:"network_id"`
-	TenantID     string `q:"tenant_id"`
-	DeviceOwner  string `q:"device_owner"`
-	MACAddress   string `q:"mac_address"`
-	ID           string `q:"id"`
-	DeviceID     string `q:"device_id"`
-	Limit        int    `q:"limit"`
-	Marker       string `q:"marker"`
-	SortKey      string `q:"sort_key"`
-	SortDir      string `q:"sort_dir"`
+	Status        string          `q:"status"`
+	Name          string          `q:"name"`
+	AdminStateUp  *bool           `q:"admin_state_up"`
+	NetworkID     string          `q:"network_id"`
+	TenantID      string          `q:"tenant_id"`
+	DeviceOwner   string          `q:"device_owner"`
+	MACAddress    string          `q:"mac_address"`
+	ID            string          `q:"id"`
+	DeviceID      string          `q:"device_id"`
+	Limit         int             `q:"limit"`
+	Marker        string          `q:"marker"`
+	SortKey       string          `q:"sort_key"`
+	SortDir       string          `q:"sort_dir"`
+	ExtraDHCPOPTS []ExtraDHCPOpts `q:"extra_dhcp_opts"`
```

Comment:
Are you sure `ExtraDHCPOpts` is a valid option for list operations? Can you verify this by providing the Neutron code that shows it or the equivalent `neutron` command?